enhancements/installer/rhel8_worker_support.md

```diff
@@ -0,0 +1,251 @@
+---
+title: rhel-8-server-worker-infra-node-support
+authors:
+  - "@mtnbikenc"
+reviewers:
+  - "@staebler"
+approvers:
+  - "@derekwaynecarr"
+creation-date: 2021-05-13
+last-updated: 2021-05-13
+status: provisional
+see-also:
+  - "https://issues.redhat.com/browse/CORS-1650"
+replaces:
+  - n/a
+superseded-by:
+  - n/a
+---
+
+# RHEL 8 Server Worker/Infra Node Support
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes to add RHEL 8 Server support for deploying worker/infra
+nodes with [openshift-ansible] RHEL8 nodes will be deployed and upgraded using
+the same Ansible playbook process however, RPM content will be delivered using
+an image in the release package.
+
+## Motivation
+
+Customers desire to deploy RHEL8 nodes and using containerized content will provide
+a better user experience than configuring RPM repositories on nodes.  Using the
+existing process will allow near-term support of RHEL8 nodes while enhanced
+longer-term RHEL8 support is developed.
+
+### Goals
+
+1. Implement an os-content image for RHEL8 RPMs
+1. Update [openshift-ansible] scaleup.yml and upgrade.yml playbooks to support RHEL8 workers
+
+### Non-Goals
+
+* Deploying/upgrading RHEL8 worker nodes through other mechanisms, i.e. operators
+* Changing the deployment/upgrade process for RHEL 7 worker nodes
```

Comment:
What is the plan for existing RHEL 7 worker nodes to be upgraded to RHEL 8? Or would they just be replaced with new RHEL 8 nodes?

Reply:
The current plan is that the customer would scale up new RHEL 8 workers and remove existing RHEL 7 workers.  We would not support a process to upgrade hosts in place.  I will add an item about not supporting OS upgrades.